utils/channel_access.py

```diff
@@ -26,7 +29,7 @@ def __init__(self, default_timeout=5, device_prefix=None):
         Constructor.
 
         :param device_prefix: the device prefix which will be added to the start of all pvs
-        :param default_timeout: the default tie out to wait for
+        :param default_timeout: the default time out to wait for
```

Comment:
Not your doing, but the documentation format here isn't Google style.

Reply:
I am not going to fix this Tom has a big refactor in the pipes